docs/Makefile

```diff
@@ -27,9 +27,13 @@ BUILDDIR      = build
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
+add:
+	@echo "Creating rST files for SYCL* Examples..."
+	py -3 rst_examples.py
+
+.PHONY: help Makefile 
 
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
-%: Makefile
+%: add Makefile
```

Comment:
in case of parallel build make may reorder targets. so, Makefile ,ay be executed before add.
it is also suggested to rename targets to get more convenient names.

Reply:
I renamed the target.

Documentation is not built with parallel builds, I don't think there is a problem.

Reply:
I can run this make with -j option. so, it may be run in parallel.

Reply:
ok, is there a way to fix this or prevent from happening?